apis/python/src/tiledbsoma/_dense_nd_array.py

```diff
@@ -3,7 +3,8 @@
 #
 # Licensed under the MIT License.
 
-"""Implementation of SOMA DenseNDArray.
+"""
+Implementation of SOMA DenseNDArray.
```

Comment:
I also prefer the newline between the triple quote and the text, but the Google guidelines for the docstrings don't have it. If we all like this style better, maybe we should make an exception but then we should make it consistent across the codebase.